src/SearchParameters/index.js

```diff
@@ -1217,9 +1216,11 @@ SearchParameters.prototype = {
    */
   getRefinedDisjunctiveFacets: function getRefinedDisjunctiveFacets() {
     // attributes used for numeric filter can also be disjunctive
-    var disjunctiveNumericRefinedFacets = intersection(
-      keys(this.numericRefinements),
-      this.disjunctiveFacets
+    var numericRefinements = Object.keys(this.numericRefinements);
+    var disjunctiveNumericRefinedFacets = this.disjunctiveFacets.filter(
+      function(facet) {
+        return numericRefinements.indexOf(facet) > -1;
+      }
     );
 
     return keys(this.disjunctiveFacetsRefinements)
```

Comment:
maybe this is done in another PR but just in case it's not.
` keys(this.disjunctiveFacetsRefinements)` can use `Object.keys` too (just like  line 1219)
```
 var numericRefinements = Object.keys(this.numericRefinements);
```

Reply:
#656